pygfx/renderers/wgpu/_wgpurenderer.py

```diff
@@ -180,10 +255,35 @@ def pixel_ratio(self, value):
                 f"Rendered.pixel_ratio expected None or number, not {value}"
             )
 
-    def render(self, scene: WorldObject, camera: Camera):
-        """Main render method, called from the canvas."""
-
-        device = self._device
+    def render(
+        self,
+        scene: WorldObject,
+        camera: Camera,
+        *,
+        region=None,
+        clear_color=None,
+        clear_depth=None,
+        flush=True,
+    ):
+        """Render a scene with the specified camera as the viewpoint.
+
+        Parameters:
+            scene (WorldObject): The scene to render, a WorldObject that
+                optionally has child objects.
+            camera (Camera): The camera object to use, which defines the
+                viewpoint and view transform.
+            region (tuple, optional): The rectangular region to draw into,
+                like the viewport, but expressed in logical coordinates.
```

Comment:
@korijn there was an open comment that I revive here: is "region" a good name for this? I want to avoid "viewport" to differentiate it from that, because this is in logical pixels. Alternatives could be "view" or "rect". Thoughts?

Reply:
Viewport is the traditional term, in opengl, threejs, and when I google it viewport is also the most common term. Let's just call it "viewport"?

I do wonder if people will go looking for a scissor test as well. Is that a thing in webgpu?

Reply:
> I don't really understand why you want to avoid the term because of the pixel type

To avoid confusion, but I guess you're right and we should just call it viewport.

> I do wonder if people will go looking for a scissor test as well. Is that a thing in webgpu?

Yes. Can add that too.

Reply:
Unless you see a good reason for it you can leave out the scissor test